### ISSUE TITLE
Set up automatic builds and image pushing

### ISSUE THREAD

**Issue by liangyuanpeng** [kubernetes-sigs/node-ipam-controller#15]
using gcr and registry.k8s.io with container image registry,same other with sig projects.
/assign

**Comment by rajibmitra** [comment]
is anyone working on this issue?

**Comment by aojea** [comment]
@liangyuanpeng seems to be working on this

**Comment by liangyuanpeng** [comment]
Yes, I am working for it, I delayed this due to work and now I am picking it up.

**Comment by mneverov** [comment]
@liangyuanpeng what's the progress here?
